From b70f1f1bc8015cea7b14bfdfc9dfde7397674a76 Mon Sep 17 00:00:00 2001
From: Michal Schmidt <mschmidt@redhat.com>
Date: Fri, 5 Sep 2014 18:35:08 -0400
Subject: [ethernet] bnx2: delete non-required instances of include <linux/init.h>

Message-id: <1409942115-26825-2-git-send-email-mschmidt@redhat.com>
Patchwork-id: 91495
O-Subject: [RHEL7.1 PATCH 1/8] bnx2: delete non-required instances of include <linux/init.h>
Bugzilla: 1092495
RH-Acked-by: Stefan Assmann <sassmann@redhat.com>
RH-Acked-by: Nikolay Aleksandrov <nikolay@redhat.com>
RH-Acked-by: John Linville <linville@redhat.com>

BZ: https://bugzilla.redhat.com/show_bug.cgi?id=1092495

part of upstream commit a81ab36bf52d0ca3a32251a923be1dbced726141
("drivers/net: delete non-required instances of include <linux/init.h>")

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/net/ethernet/broadcom/bnx2.c b/drivers/net/ethernet/broadcom/bnx2.c
index 22e8001..6c9e1c9 100644
--- a/drivers/net/ethernet/broadcom/bnx2.c
+++ b/drivers/net/ethernet/broadcom/bnx2.c
@@ -23,7 +23,6 @@
 #include <linux/vmalloc.h>
 #include <linux/interrupt.h>
 #include <linux/pci.h>
-#include <linux/init.h>
 #include <linux/netdevice.h>
 #include <linux/etherdevice.h>
 #include <linux/skbuff.h>
-- 
1.7.1